scripts/push_docs_to_branch.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+set -Eeuo pipefail
+
+RASA_FOLDER=rasa
+TESTS_FOLDER=tests
+DATA_FOLDER=data
+TMP_DOCS_FOLDER=tmp-documentation
+TODAY=`date "+%Y%m%d"`
+# we build new versions only for majors
+PATTERN_FOR_NEW_VERSION="^refs/tags/[0-9]+\\.0\\.0$"
+MASTER_REF=refs/heads/master
+
+[[ ! $GITHUB_REF =~ $PATTERN_FOR_NEW_VERSION ]] \
```

Comment:
I am not sure this works as we'd want it to:
- we want to have separate documentation pages for the separate minor versions (e.g. one for 2.0 and another one for 2.1 and one for master/next)
- on a push to the release branch e.g. `2.1.x`, we need to update the `2.1` documentation (same for master, but that already works)

Reply:
- I thought we discussed we wanted the docs only for majors; will update the regex and it will take care of it
- I totally missed this part of the workflow :/

Reply:
@tmbo something I am wondering: instead of listening to a push on a release branch + finding out the version "manually", why not listening to the push of the patch tag (which is done from the release branch), and modify the script to do a special case for this one? It would simplify the implementation I think and we wouldn't lose any functionality.

Reply:
456caef223d